app/assets/javascripts/components/common/list.jsx

```diff
@@ -19,18 +19,19 @@ const List = createReactClass({
   },
 
   componentDidMount() {
-    if ($('.persist-area').length !== 0) {
-      let clonedHeaderRow;
-      $(".persist-area").each(function () {
-          clonedHeaderRow = $(".persist-header", this);
-          clonedHeaderRow
-            .before(clonedHeaderRow.clone())
-            .css("width", clonedHeaderRow.width())
-            .addClass("floatingHeader");
-      });
+    const persistAreaElements = document.getElementsByClassName('persist-area');
+    if (persistAreaElements.length !== 0) {
+      for (let i = 0; i < persistAreaElements.length; i++) {
+        const persistHeaderRow = persistAreaElements[i].getElementsByClassName('persist-header');
+        const clonedHeaderRow = persistHeaderRow[0].cloneNode(true);
+        const style = window.getComputedStyle(persistHeaderRow[0]);
+        const width = style.getPropertyValue('width');
+        clonedHeaderRow.style.width = width;
+        clonedHeaderRow.classList.add("floatingHeader");
+        persistAreaElements[i].childNodes[0].insertBefore(clonedHeaderRow, persistHeaderRow[0]);
+      }
```

Comment:
@ragesoss Is this what you were suggesting? I have refactored some of the code but I don't see a point why not to use JQuery!!